rpadmin/api_migration.go

```diff
@@ -53,16 +61,22 @@ func (a *AdminAPI) DeleteMigration(ctx context.Context, id int) error {
 	return a.sendAny(ctx, http.MethodDelete, fmt.Sprintf("baseMigrationEndpoint%d", id), nil, nil)
 }
 
-// ExecuteMigration executes a specific action on a migration identified by its ID. The action must be one of:
-// prepare, execute, finish, cancel.
-func (a *AdminAPI) ExecuteMigration(ctx context.Context, id int, action string) error {
-	validActions := map[string]bool{
-		"prepare": true,
-		"execute": true,
-		"finish":  true,
-		"cancel":  true,
-	}
-	if !validActions[action] {
+type MigrationAction int
+
+const (
+	PrepareAction MigrationAction = iota
+	ExecuteAction
+	FinishAction
+	CancelAction
```

Comment:
Can we `/s/Action/MigrationAction` 
This is an exported constant of the admin package and the name might be too generic, to our callers this looks like this:
```
rpamin.ExecuteAction
```

But is not clear what type of action in the whole admin API.